app/controllers/accountants_controller.rb

```diff
@@ -1,10 +1,31 @@
 # Controller pertaining to accountant functions, usually fulfillments.
 class AccountantsController < ApplicationController
-  def index; end
+  before_action :confirm_admin_user
+  before_action :find_patient, only: [:edit]
+
+  def index
+    fulfilled_pts = Patient.where('fulfillment.fulfilled' => true)
```

Comment:
note to self to scope this a little tighter so we don't light up the db by accident

Reply:
done